docs: Make Manual File Creation the default tab (#7305)
spectrocloud/librarium: docs/docs-content/clusters/edge/edgeforge-workflow/prepare-user-data.md

[Installer Reference](../edge-configuration/installer-reference.md) resource to learn more about all the supported
configuration parameters you can use in the configuration user data.

:::preview

The [Appliance Studio](../../../deployment-modes/appliance-mode//appliance-studio.md) is a Tech Preview feature and is
subject to change. Do not use it to create files for production workloads. Both `.arg` and `user-data` are plain text

> files. Once you are familiar with the parameters, you can manually create and edit them.

Reviewer: It's not that important, but the last two sentences seem out of place within the context of the tech preview statement.

Maybe take them out of this admonition and place them somewhere before it?

Author: ```suggestion
The [Appliance Studio](../../../deployment-modes/appliance-mode//appliance-studio.md) is a Tech Preview feature and is subject to change. Do not use it for production workloads. Instead, create the `.arg` and `user-data` files manually.
```

 I was worried it might seem like a strange change of subject. Now I see it does look like that indeed. 
 How about the option above, do you think it's better? I don't think it's necessary to mention the files are plain text in that context.

Reviewer: ```suggestion
The [Appliance Studio](../../../deployment-modes/appliance-mode//appliance-studio.md) is a Tech Preview feature and is
subject to change. For production workloads, create the `.arg` and `user-data` files manually.
```
